api/voters/models.py

```diff
@@ -40,6 +42,19 @@ def __str__(self):
     def name(self):
         return f"{self.first_name} {self.last_name}"
 
+    def update_user(self):
+        self.user, created = User.objects.get_or_create(email=self.email)
+        if created:
+            self.user.username = self.email
+            self.user.password = "<unset>"
```

Comment:
Because this is unencrypted, the user will be unable to log in until this is set properly.